api/app/controllers/spree/api/v2/storefront/products_controller.rb

```diff
@@ -33,6 +33,13 @@ def resource_serializer
             Spree::Api::Dependencies.storefront_product_serializer.constantize
           end
 
+          def scope
+            model_class.
+              available_in_currency(currency: current_currency).
```

Comment:
```suggestion
              active(current_currency).
```

we don't need new scope

Reply:
@damianlegawiec `active` scope breaking this part of spec: https://app.circleci.com/pipelines/github/spree/spree/1738/workflows/e4be4eb3-4acc-48f5-a3cd-b8cd47352c6e/jobs/20831 

should I pass to `not_discontinued` scope boolean flag relay on incomming params ?